app/models/division.rb

```diff
@@ -114,4 +114,20 @@ def locale_names
       I18n.t("locale_name.#{locale}", locale: locale)
     end
   end
+
+  def quickbooks_connected?
+    self.class.connector.connected?
+  end
+
+  def quickbooks_connect(access_token:, params:)
+    self.class.connector.connect(access_token: access_token, params: params)
+  end
+
+  def quickbooks_disconnect
+    self.class.connector.disconnect
+  end
```

Comment:
Having all these methods prefixed with `quickbooks_` is a code smell IMO. It says that there is a bunch of functionality that should be in its own class...

BUT looking more closely really it seems like what these are are just delegations to follow LoD, which is good!

So what I recommend is:

1. Making `connector` an instance method and renaming it to `quickbooks` (right now you're setting an instance var in a class method which I don't even know how that's working!)
1. Using `delegate ..., prefix: true` to setup your delegators. Then you can delete all these methods.